src/modeler/qubot.rs

```diff
@@ -311,7 +321,7 @@ impl<'a> Qubot<'a> {
         self.mapping_carries.insert(key, qubit_carry);
     }
 
-    pub fn query_existence(&self, gate: &GateRef) -> Option<QubitRef> {
+    pub fn _query_existence(&self, gate: &GateRef) -> Option<QubitRef> {
```

Comment:
I think it's OK to remove this unused function entirely unless you are planning to use it in a follow-up change.